site/content/user-guide/jmeter_load_tests.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "JMeter Load Tests"
+weight = 300
```

Comment:
Okay - changed the weights from user-guide and dev-guide to match also the order for the forward/back buttons